src/Web/Twitter/Feed.hs

```diff
@@ -43,13 +43,22 @@ timeline oauth credential count excludeReplies username = do
       Right ts     -> Right $ map (simplifyTweet . linkifyTweet) ts
 
 createRequest :: String -> Int -> Bool -> IO Request
+#if MIN_VERSION_http_conduit(2,1,11)
+createRequest username count excludeReplies = parseUrlThrow $ timelineUrl username count excludeReplies
+#else
 createRequest username count excludeReplies = parseUrl $ timelineUrl username count excludeReplies
+#endif
 
 getResponse :: OAuth -> Credential -> Request -> IO (Response BS.ByteString)
 getResponse oauth credential req =
+#if MIN_VERSION_http_conduit(2,1,11)
+  do
+  m <- newManager tlsManagerSettings
```

Comment:
@CristhianMotoche Do we really need this one? `newManager` looks like it's there since 0.10, so I guess it would just work if we just stop using `withManager`. I didn't test this, though.